great-idea/index.html

```diff
@@ -17,7 +17,75 @@
   <body>
 
     <!-- Copy and paste your HTML from the first UI project here -->
-      
     
+    <header>
+      <div class='header-top'>
+        <nav>
+          <a href='services.html'>Services</a>
+          <a href=''>Product</a>
+          <a href=''>Vision</a>
+          <a href=''>Features</a>
+          <a href=''>About</a>
+          <a href=''>Contact</a>
+        </nav>
+      
+        <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
```

Comment:
This logo image could be included within the nav tag, and that nav could be turned into a flex-box. That way, the image will be included in the flow with all the a tags at different window sizes. Something to think about.